decentraland/unity-explorer
# Fix/UI sorting

- Ignore re-parenting of the root UI transform (it looks like an SDK bug/vulnerability)
- Fix re-parenting to the root upon a component change: converted the root to the fully-fledged `UITransformComponent` with children, sorting, etc.
- Work-around very strange unsorted UI Elements: some `UITransform` may have default `0` `rightOf`, looks like an SDK bug.
- Changed sorting to the node-based scheme:
   - Linear-based one worked as expected due to the coincidence: entities are created in the sorted order at first place
   - It would not work if `rightOf` is changed dynamically
- Relax media hosts check, see https://github.com/decentraland/unity-renderer/pull/5844 
- Fix repetitive teleport adding

----
How to QA:
- Try Goerli-plaza: `(80,-3)` should work as expected: with proper layout and order, all textures should be loaded in the dialogs
- Genesis City: Fix #1175 
- MetadyneLabs: Fix #931 
- If you have other UI-rich scenes in mind try them
